bsp/stm32/stm32f411-st-nucleo/applications/arduino_pinout/pins_arduino.c

```diff
@@ -46,3 +46,29 @@ const pin_map_t pin_map_table[]=
     {A6, RT_NULL, "adc1", RT_ADC_INTERN_CH_VREF},   /* ADC, On-Chip: internal reference voltage */
     {A7, RT_NULL, "adc1", RT_ADC_INTERN_CH_TEMPER}, /* ADC, On-Chip: internal temperature sensor */
 };
+
+void pins_switch_to_spi(const char *bus_name)
+{
+    if(!rt_strcmp(bus_name, RTDUINO_DEFAULT_SPI_BUS_NAME))
```

Comment:
RTDUINO_DEFAULT_SPI_BUS_NAME 换成具体的spi设备
```suggestion
    if(!rt_strcmp(bus_name, "spi1"))
```